pages/Nightvision-goggles.md

```diff
@@ -0,0 +1,7 @@
+The Night Vision Goggle is a useful [Technical Gadget](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Technical-Gadget) which allows you to see in the Night or in Caves.
```

Comment:
The file name should be "Night-Vision-Goggles", you even made a link to the correct spelling in your category .md